enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk/index.ts

```diff
@@ -0,0 +1,6 @@
+import { PLUGIN_EMBEDDING_IFRAME_SDK } from "metabase/plugins";
+import { hasPremiumFeature } from "metabase-enterprise/settings";
+
+if (hasPremiumFeature("embedding_iframe_sdk")) {
+  PLUGIN_EMBEDDING_IFRAME_SDK.isEnabled = () => true;
+}
```

Comment:
Will be used when the UI is introduced. For now it's mostly unused as we detect with `hasPremiumFeature` directly to avoid importing EE plugins into the isolated route.